### ISSUE TITLE
rollups failing at LazyLoad

### ISSUE THREAD

**Issue by gvaish** [YahooArchive/mojito#435]
When I run `mojito compile rollups`, I get the following error:

```
[info] MojitoServer: ALREADY EXISTS: asset favicon.ico -- in mojit shared -- using from pkg todomvc@0.0.1 -- skipping from pkg mojito@0.3.30
Creating rollups...

node.js:201
        throw e; // process.nextTick error, or 'error' event on first tick
              ^
Error: EACCES, permission denied '/usr/local/lib/node_modules/mojito/lib/app/mojits/LazyLoad/rollup.client.js'
    at Object.<anonymous> (fs.js:230:18)
    at Object.openSync (/usr/local/lib/node_modules/mojito/node_modules/glob/node_modules/graceful-fs/graceful-fs.js:69:28)
    at Object.writeFileSync (fs.js:618:15)
    at rollOneUp (/usr/local/lib/node_modules/mojito/lib/management/commands/compile.js:442:12)
    at Object.rollups (/usr/local/lib/node_modules/mojito/lib/management/commands/compile.js:472:17)
    at Object.run (/usr/local/lib/node_modules/mojito/lib/management/commands/compile.js:156:17)
    at main (/usr/local/lib/node_modules/mojito/lib/management/cli.js:124:13)
    at Object.<anonymous> (/usr/local/lib/node_modules/mojito/lib/management/cli.js:136:1)
    at Module._compile (module.js:444:26)
    at Object..js (module.js:462:10)
```

I ran it on https://github.com/gvaish/todomvc/tree/dev-gvaish/architecture-examples/mojito/src

**Comment by drewfish** [comment]
This looks like a bug in Mojito. (At one point, it had a check to skip making the rollups for the mojits that ship with Mojito, for exactly the filesystem issues you ran into.)

What version of Mojito are you using? This is hopefully fixed in that latest version (0.4.2).

**Comment by gvaish** [comment]
Seems fixed in 0.4.42. Was earlier running 0.3.30
